docs/tools.rst

```diff
@@ -65,6 +74,16 @@ with GitHub_'s API. It allows one to turn issues on GitHub into
 pull-requests. If that is done then once the pull-request is merged into
 the main branch the issue is automatically closed.
 
+
+We use the `Git Flow`_ branching model as part of our development.
+It's a convenient way to manage your branches. You are not required to use
+Git Flow but you are required to follow naming conventions it sets
+with regard to branch-names and prefixes.
+
+with GitHub_'s API. It allows one to turn issues on GitHub into
```

Comment:
Not sure this paragaph belongs here? The start is unconnected to anything and the rest seems to be about HubFlow maybe?